rig/routing_table/ordered_covering.py

```diff
@@ -738,7 +755,7 @@ def _refine_downcheck(merge, aliases, min_goodness):
 
                 # Add this settable mask and the required values to the
                 # settables list.
-                bits_and_vals.update((bit, bool(key & bit)) for bit in
+                bits_and_vals.update((bit, not bool(key & bit)) for bit in
```

Comment:
The "bool" is superfluous here due to the "not", but up to you as you may prefer it as a stylistic choice.